Resources/Prototypes/Entities/Structures/Power/chargers.yml

```diff
@@ -53,6 +53,8 @@
         components:
         - HitscanBatteryAmmoProvider
```

Comment:
Has this been tested to see if laser guns can go in the recharger? IIRC they should not and should use wall rechargers.